<launch>
    <arg name='robot_idx' default="1" />
    <node pkg="robot_worker" type="robot_worker_node" name="robot_worker_node" ns='robot_$(arg robot_idx)' output="screen">
        <param name='robot_idx' value = '$(arg robot_idx)'/>
    </node>
</launch>